omod-1.12/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_12/OrderGroupResource1_12.java

```diff
@@ -0,0 +1,229 @@
+		if (rep instanceof DefaultRepresentation || rep instanceof FullRepresentation) {
+			modelImpl
+			        .property("uuid", new StringProperty())
+			        .property("display", new StringProperty())
+			        .property("voided", new BooleanProperty())
+			        .property("patient", new RefProperty("#/definitions/PatientGetRef"))
+			        .property("encounter", new RefProperty("#/definitions/EncounterGetRef"))
+			        .property("orders", new RefProperty("#/definitions/OrderGetRef"))
+			        .property("orderSet", new RefProperty("#/definitions/OrdersetGetRef"));
+			
+		}
+		if (rep instanceof DefaultRepresentation) {
+			modelImpl
+			        .property("uuid", new StringProperty())
+			        .property("display", new StringProperty())
+			        .property("voided", new BooleanProperty())
+			        .property("patient", new RefProperty("#/definitions/PatientGetRef"))
+			        .property("encounter", new RefProperty("#/definitions/EncounterGetRef"))
+			        .property("orders", new RefProperty("#/definitions/OrderGetRef"))
+			        .property("orderSet", new RefProperty("#/definitions/OrdersetGetRef"));
+			
+		} else if (rep instanceof FullRepresentation) {
+			modelImpl
+			        .property("uuid", new StringProperty())
+			        .property("display", new StringProperty())
+			        .property("voided", new BooleanProperty())
+			        .property("auditInfo", new BooleanProperty())
+			        .property("patient", new RefProperty("#/definitions/PatientGetRef"))
+			        .property("encounter", new RefProperty("#/definitions/EncounterGetRef"))
+			        .property("orders", new ArrayProperty(new RefProperty("#/definitions/OrderGetRef")))
+			        .property("orderSet", new RefProperty("#/definitions/OrdersetGetRef"));
+		}
+		else if (rep instanceof RefRepresentation) {
+			modelImpl
+			        .property("display", new StringProperty())
+			        .property("uuid", new StringProperty());
+			
+		}
+		return modelImpl;
+	}
+	
+	@Override
+	public Model getCREATEModel(Representation representation) {
+		ModelImpl model = new ModelImpl()
+		        .property("patient", new StringProperty().example("uuid"))
+		        .property("encounter", new StringProperty().example("uuid"))
+		        .property("orders", new ArrayProperty(new RefProperty("#/definitions/OrderCreate")))
+		        .property("orderSet", new StringProperty().example("uuid"));
+		return model;
+	}
+	
+	@Override
+	public Model getUPDATEModel(Representation rep) {
+		ModelImpl model = new ModelImpl()
+		        .property("orders", new ArrayProperty(new RefProperty("#/definitions/OrderCreate")));
+		return model;
+	}
+	
+	@PropertySetter("orders")
+	public static void setGroupMembers(OrderGroup orderGroup, List<Order> members) {
+		if(members != null && members.size() >0) {
+		for (Order member : members) {
+			orderGroup.addOrder(member);
+			member.setOrderGroup(orderGroup);
+		}
+		}
+	}
+	
+	@PropertyGetter("orders")
```

Comment:
What is the use of the property setter and getter annotations for orders? Are there any tests which will fail when you remove them?

Reply:
oh let me take closer look at it ,your right the default setters and getters in the orderGroup class already do what am repeating here. let me test it out and update the Pr

Reply:
yes , that was just an unnecesary duplication . the default setter and getter in the OrderGroup class already can do it. i hadnt looked at it keenly